tiledb/CMakeLists.txt

```diff
@@ -357,6 +357,8 @@ add_custom_target(
   COMMENT "Re-generate ${MGC_GZIPPED_BIN_PATH3} for embedded magic.mgc support"
 )
 
+add_dependencies(TILEDB_CORE_OBJECTS exercise_link_webp_test)
```

Comment:
This dependency does not look right (at very least the dependency should be on a library target, not on a test executable).